.github/workflows/run-security-scans.yml

```diff
@@ -189,26 +205,68 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set high and critical severity outputs
+        id: severity_check_outputs
+        run: |
+          severity_check_high=$(curl -s -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?ref=${{github.ref}} | jq '[.[] | select(.state == "open" and (.rule.security_severity_level == "high"))] | any') 
+          severity_check_critical=$(curl -s -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?ref=${{github.ref}} | jq '[.[] | select(.state == "open" and (.rule.security_severity_level == "critical"))] | any')
+          echo "severity_check_high=$severity_check_high" >> $GITHUB_OUTPUT
+          echo "severity_check_critical=$severity_check_critical" >> $GITHUB_OUTPUT
+
       # Returns 'true' if there is Code Scanning alert with 'high' or 'critical' security severity level
-      - name: Check for Critical and High Severity
+      - name: Succeed or fail based on severity
         id: severity_check
         run: |
-          severity_check=$(curl -s -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?ref=${{github.ref}} | jq '[.[] | select(.state == "open" and (.rule.security_severity_level == "high" or .rule.security_severity_level == "critical"))] | any') 
```

Comment:
En annen, ikke relatert, svakhet er at man ikke sjekker på vellykket status til HTTP-kall. Dette vil gjøre at sjekken kan gå grønt uten at kritiske sårbarheter vil blokkere workflow!

Gruner til at man får en ikke vellykket response kan være feil i credentials. En annet scenarie er at GitHub Advanced Security ikke er slått på.

For mer informasjon, se https://docs.github.com/en/rest/code-scanning?apiVersion=2022-11-28#list-code-scanning-alerts-for-a-repository

Reply:
Opprettet en bugsak på denne sårbarheten, se https://kartverket.atlassian.net/browse/SKIP-914